resources/assets/coffee/_classes/forum.coffee

```diff
@@ -185,6 +186,13 @@ class @Forum
 
     # using jquery smooth scrollTo will cause unwanted events to trigger on the way down.
     window.scrollTo window.pageXOffset, postTop
+    @highlightPost post
+
+
+  highlightPost: (post) =>
+    @highlightedPost?.classList.remove 'js-forum-post--highlighted'
```

Comment:
looks like it won't work quite right on navigation. Maybe just select all with that class instead?